frontend/src/mocks/fixtures/api/projects/:team_id/insights/trendsLine.json

```diff
@@ -4,37 +4,22 @@
     "name": "",
     "derived_name": "Pageview count",
     "filters": {
+        "display": "ActionsLineGraph",
         "events": [
             {
                 "id": "$pageview",
-                "name": "$pageview",
-                "type": "events",
-                "order": 0
-            },
-            {
-                "id": "$geoip_country_name",
-                "name": "$geoip_country_name",
                 "type": "events",
-                "order": 0
+                "order": 0,
+                "name": "$pageview",
+                "custom_name": null,
+                "math": null,
+                "math_property": null,
+                "math_group_type_index": null,
+                "properties": {}
             }
```

Comment:
Hmm, we do still need a second series here though, as we need to have at least one snapshot covering this case (as we discovered in https://github.com/PostHog/posthog/pull/17580)

Reply:
Makes sense. I've added a separate mock with multiple series.